src/gc/env/gcenv.os.h

```diff
@@ -7,6 +7,17 @@
 #ifndef __GCENV_OS_H__
 #define __GCENV_OS_H__
 
+#ifdef Sleep
+// This is a funny workaround for the fact that "common.h" defines Sleep to be
```

Comment:
This is CoreCLR specific workaround - can it be somewhere under VM instead?

Also, this hack made sense for full .NET Framework. It is not really valuable for CoreCLR. If it is a problem to move it to somewhere under VM, I would rather delete all the code to `#define Sleep` to prevent people from using it.